bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 13.0.4
+version: 13.0.5
```

As it's a new feature more than a fix, can you bump the minor version instead of the patch?
```suggestion
version: 13.1.0
```